### PR TITLE
Improved events query.

### PR DESCRIPTION
**Pull request by nikomakela**
PT-797 Removed pEvent and venue fields (but left an id and nextOccurrenceDatetime) to much improve the events query. Single event query is unchanged.

API changes done in https://github.com/City-of-Helsinki/palvelutarjotin/pull/129.

### CODE REVIEW

**Comment by nikomakela** [comment]
I first had changes to 1 query file only. It passed testing and linter on my machine, but Github was more demanding :) Needed to add typing here and there.

**Comment by nikomakela** [comment]
Pull request https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/80 might overlap with this quite badly. This branch should be checked against it.

**Comment by nikomakela** [comment]
In a refinement session it was said that this PR needs to be deployed this week before the next week's demo session.

File: src/domain/event/eventKeywords/EventKeywords.tsx [hunk 1]
```diff
@@ -1,14 +1,17 @@
 import React, { ReactElement } from 'react';
 
-import { EventFieldsFragment } from '../../../generated/graphql';
+import {
+  EventFieldsFragment,
+  EventsFieldsFragment,
```
Comment (Sartonon) [inline src/domain/event/eventKeywords/EventKeywords.tsx:5]: Hmm this is the downside passing whole event object as a prop and trying to add types 🤔 Hopefully we don't need more of these field fragments in the future :)

**Review by Sartonon — APPROVED** [review]
Looks good.

File: src/domain/event/eventKeywords/EventKeywords.tsx (hunk 1 above)
Comment (nikomakela) [reply src/domain/event/eventKeywords/EventKeywords.tsx:5]: True. Otherwise I would have managed with 1 file change.

**Pull request merged**